csharp/test/Ice/proxy/MyDerivedClassI.cs

```diff
@@ -23,6 +27,21 @@ public bool IceIsA(string typeId, Current current, CancellationToken cancel)
             return typeof(IMyDerivedClass).GetAllIceTypeIds().Contains(typeId);
         }
 
-        private Dictionary<string, string>? _ctx;
+        public IRelativeTestPrx OpRelative(ICallbackPrx callback, Current current, CancellationToken cancel)
+        {
+            TestHelper.Assert(callback.IsFixed);
+            IRelativeTestPrx relativeTest =
+                current.Adapter.AddWithUUID(new RelativeTest(), IRelativeTestPrx.Factory).Clone(relative: true);
```

Comment:
Should/could the proxies created by the OA be relative by default?

Reply:
It could make sense for ice2 endpoint-less object adapters to create relative proxies by default, as they are used primarily for bidir callbacks.

The downside is this wouldn't be a uniform rule:
- ice2 endpoint-less object adapter => creates relative=true proxies
- other object adapter => creates non-relative proxies 

So overall, I think the current simple behavior (all OAs create non-relative proxies) is better.